### PR TITLE
PAL60 improvements: Allow setting in GameINI, disable for NTSC games, store in Movie files.

### PR DESCRIPTION
**Pull request by AdmiralCurtiss**
This is the PAL60-related improvements of #2532 without the GUI changes, which I hope makes it easier to test and merge.

Fixes issues [7847](https://code.google.com/p/dolphin-emu/issues/detail?id=7847), [7714](https://code.google.com/p/dolphin-emu/issues/detail?id=7714), and [8036](https://code.google.com/p/dolphin-emu/issues/detail?id=8036). Also adds the PAL60 option to the Movie file header.

### CODE REVIEW

**Comment by lioncash** [comment]
@dolphin-emu-bot rebuild

**Comment by JosJuice** [comment]
@dolphin-emu-bot rebuild

**Comment by JosJuice** [comment]
LGTM

File: Source/Core/Core/Movie.h [hunk 1]
```diff
@@ -101,7 +101,8 @@ struct DTMHeader
 	u8   bongos;
 	bool bSyncGPU;
 	bool bNetPlay;
-	u8   reserved[13];      // Padding for any new config options
+	bool bPAL60;
+	u8   reserved[12];      // Padding for any new config options
```
Comment (phire) [inline Source/Core/Core/Movie.h]: Slightly off-topic. We are relying on sizeof(bool) to be 1, while in the c++ standard it is implementation defined.

Comment (BhaaLseN) [inline Source/Core/Core/Movie.h]: Even so, this has been there before and probably doesn't belong in this PR anyways. Maybe drop an Issue to keep track of this?

**Comment by JosJuice** [comment]
@dolphin-emu-bot rebuild
